scripts/dorelease.py

```diff
@@ -241,6 +241,21 @@ def print_appveyor_urls():
                                    "artifacts", a["fileName"]))
 
 
+def check_python_command():
+    cmd = "python"
+    py_version = str(check_output([cmd, "--version"], stderr=STDOUT))
+    if " 3." in py_version:
+        return cmd
+    cmd = "python3"
+    try:
+        ok = call([cmd, "--version"])
+        if ok == 0:
+            return cmd
+    except FileNotFoundError:
+        cmd = None
+    return cmd
```

Comment:
Since we don't support python2 at all anymore, is this necessary?  We can just call `setup.py` with `python3` in `main` and if it's not available it wont work.

Reply:
You're right if  there will always be a python3 command even when no python2 is installed. Is that the case?

Reply:
Yes, `python 3.x` (any version) should always be available as `python3`.